client/my-sites/upgrades/checkout/thank-you.jsx

```diff
@@ -63,6 +66,14 @@ var CheckoutThankYou = React.createClass( {
 	},
 
 	componentDidMount: function() {
+		var selectedSite = this.props.lastTransaction.selectedSite;
+		this.props.refreshSitePlans( selectedSite.ID );
+
+		// refresh the list of sites to update the `site.plan` property.
+		Dispatcher.handleServerAction( {
```

Comment:
We should use `Dispatcher.handleViewAction` because this action is triggered by the client.